react-web/src/components/AvatarDropdown/AvatarDropdown.tsx

```diff
@@ -7,13 +7,25 @@ import Menu from "@mui/material/Menu";
 import MenuItem from "@mui/material/MenuItem";
 import Avatar from "@mui/material/Avatar";
 import "./AvatarDropDown.scss";
+import useLocalStorage from "hooks/useLocalStorage";
 
 const AvatarDropDown = () => {
   const navigate = useNavigate();
   const dispatch = useAppDispatch();
   const logOut = () => {
     dispatch(logout());
   };
+  const [, setIsLoggedIn] = useLocalStorage(
+    "isLoggedIn",
+    localStorage.getItem("isLoggedIn") === "true" ? true : false
+  );
+
+  const [, setUserDetails] = useLocalStorage(
+    "userDetails",
```

Comment:
same for this

Reply:
https://github.com/input-output-hk/dapps-certification/pull/78#discussion_r1281488595

Reply:
Done